### PR TITLE
Byte can be simplified to byte

### PR DESCRIPTION
**Pull request by hassanhabib**
## Simplify Byte

Using ``` byte ``` instead of ``` Byte ```.

### CODE REVIEW

File: docs/csharp/programming-guide/types/index.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Value types and reference types in the CTS
   
 ```csharp  
 // Static method on type Byte.
```
Comment (rpetrusha) [inline docs/csharp/programming-guide/types/index.md]:
```suggestion
// Static method on type byte.  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for contributing to the dotnet/docs repo, @hassanhabib. If Byte is changed to byte in the variable declaration, it should also be changed in the comment.

File: docs/csharp/programming-guide/types/index.md (hunk 1 above)
Comment (hassanhabib) [reply docs/csharp/programming-guide/types/index.md]: Good catch! thanks. fixed.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thank you for making the additional change, @hassanhabib. We'll merge your PR now.

**Pull request merged**